portfolio/src/main/webapp/index.html

```diff
@@ -7,19 +7,133 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>Emily's Portfolio</h1>
-      <img src=/images/headshot.jpg alt="My headshot" width="220" height="300">
-      <p>My name is Emily Vera and I am a rising junior studying computer science at the
-          University of Illinois at Urbana-Champaign. In addition to computers, I am also 
-          heavily interested in entreprenuership and the business side of things, and am 
-          minoring in technology and management. I am in a social sorority: Phi Sigma Sigma,
-          and am also involved in other clubs such as Society of Women Engineers, Girls
-          Who Code, and am currently a STEP Intern at Google!</p>
-      <p>This is my portfolio. Have a look around!</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="header">
+        <div class="header-right">
+            <a class="active" href="#home">Home</a>
+            <a href="#about">About</a>
+            <a href="#socials">Socials</a>
+            <a href="#overview">Overview</a>
+            <a href="#projects">Projects</a>
+        </div>
+    </div>
+    <img class="logo" src="/images/logo.png" alt="My logo" width="70" height="70">
+    <h1 class="center">Student, Tech Enthusiast & Human</h1>
+    <h3 class="center">Welcome to my portfolio. Have a look around!</h3>
+    <img src="/images/bitmoji.png" alt="My headshot" width="315" height="280" class="bitmoji-headshot">
+    <img src="/images/tech.png" alt="Tech clipart" width="300" height="250" class="center">   
+    <a id="about"></a>
+    <div class="background-block"> 
+        <div class="row">
+            <div class="two-column-left">
+                <img src="/images/headshot.jpg" alt="My headshot" width="400" height="500" class="real-headshot"> 
+            </div>
+            <div class="two-column-right">
+                <h2 class="aboutme-block">Hi, I'm Emily. Nice to meet you.</h2>
+                <p class="white aboutme-block">My name is Emily Vera and I am a rising junior studying 
+                computer science at the University of Illinois at Urbana-Champaign. In addition to 
+                computers, I am also heavily interested in entreprenuership and the business side of 
+                things, and am minoring in technology and management. I am in a social sorority: 
+                Phi Sigma Sigma, and am also involved in other clubs such as Society of Women Engineers, 
+                Girls Who Code, and am currently a STEP Intern at Google!</p>
```

Comment:
You can take out optional tags. I believe the &lt;/p&gt; tag is optional.

https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

Reply:
I think the linter might actually complain about not having end tag.